common/djangoapps/django_comment_common/signals.py

```diff
@@ -0,0 +1,12 @@
+"""Signals related to the comments service."""
+
+from django.dispatch import Signal
+
+
+thread_created = Signal(providing_args=['user'])
+thread_edited = Signal(providing_args=['user'])
+thread_voted = Signal(providing_args=['user'])
+comment_created = Signal(providing_args=['user'])
```

Comment:
I think I've come around to all the individual signals-- it does make the code much more self-documenting and is better for testing.